packages/styles/settings-tools/_s.fonts-faces.scss

```diff
@@ -18,12 +17,12 @@ $font-family: if($presetAdeo, "Roboto", "LeroyMerlin");
 $font-weights: (
   "light": 300,
   "regular": 400,
-  "semi-bold": 600,
+  "semi-bold": if($preset-adeo, 700, 600)
 );
 
 @mixin import-font-families($italic: true) {
   @font-face {
-    font-family: "LeroyMerlin";
+    font-family: LeroyMerlin;
```

Comment:
The inverted commas should be kept when using a font that does not belong to the browser.